tests/test_encoding.py

```diff
@@ -88,6 +89,300 @@ def test_encode_coded_const_infer_order(self) -> None:
         )
         self.assertEqual(req.encode(), bytearray([0x7D, 0xAB]))
 
+    def test_string_encodings(self) -> None:
+        # ISO 8859-1 (latin1)
+        encode_state = EncodeState()
+        encode_state.emplace_atomic_value(
+            internal_value="ä?a",
```

Comment:
What's this?

Reply:
this is a test case containing a non-ASCII character (`ä`. mainly because it was conveniently accessible on my keyboard.)